### ISSUE TITLE
Access-Control-Allow-Origin

### ISSUE THREAD

**Issue by Samuel-Momoh** [realArafatBen/nigeria-state-city-api#2]
Access to XMLHttpRequest at 'https://locus.fkkas.com/api/' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

**Comment by realArafatBen** [comment]
You can try it now, 
i have updated the api to allow cors. 
Any other issues let me know. thanks

**Comment by Samuel-Momoh** [comment]
you might need to pass a wild card for cor configuration Access-Control-Allow-Origin: '*'. Issue still persist

**Comment by realArafatBen** [comment]
That's already done.  
Basically using the API suppose not to bring up this issue. 
Can u share your code?

**Comment by Samuel-Momoh** [comment]
you can close the issue, switching to javascript fetch seems fix the problem

**Comment by realArafatBen** [comment]
Alright. 
thanks
